src/main/resources/assets/styles/_statisticFigures.scss

```diff
@@ -0,0 +1,121 @@
+.statistic-figures {
+  background-color: white;
+  padding: 1.5rem 2.5rem 4.5rem;
+  border-radius: 2rem;
+
+  .title-wrapper {
+    margin-bottom: 2rem;
+    margin-top: 1.5rem;
+    align-items: center;
+    display: flex;
+
+    h2 {
+      margin-bottom: 0;
+      margin-right: 0.5rem;
+    }
+
+    .icon-wrapper {
+      margin-left: auto;
+    }
+  }
+
+  .part-accordion {
+    padding: 10px;
```

Comment:
There are a few places in this file where `padding` and `margin` can be converted to `rem`.